Other/RevontheusSet.cs

```diff
@@ -0,0 +1,51 @@
+//cs_include Scripts/CoreBots.cs
+
+using Skua.Core.Interfaces;
+using Skua.Core.Options;
+
+public class RevontheusSet
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+
+    public List<IOption> Options = new List<IOption>()
+    {
+        CoreBots.Instance.SkipOptions,
+        new Option<bool>("Equip", "Equip the revontheus set?", "", false),
+    };
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        GetSet();
+
+        Core.SetOptions(false);
+    }
+
+    public void GetSet()
+    {   
+        string[] Set = {
+            "Revontheus Visage",
+            "Wings of Revontheus",
+            "Revontheus"
+        };
+
+        if(Core.CheckInventory(Set)){
+            Core.Logger("You already have the set");
+            return;
+        }
+        
+        Core.AddDrop(Set);
+        Core.EquipClass(ClassType.Solo);
+        
+        while(!Bot.ShouldExit && !Core.CheckInventory(Set)) {
+            Core.KillMonster("underworld", "r10", "left", "Undead Legend", log:false);
+        }
+
+        if(Bot.Config.Get<bool>("Equip"))
+            Core.Equip(Set);
+
+        Core.Logger("Finished Farming", stopBot: true);
```

Comment:
this is redundant already bot log bot stopped and the user will know as he will be wearing the set also add else statement for not wearing to add stuff in bank

Reply:
these are not AC tagged, should i still proceed ?